### PR TITLE
260-restructure-i18n-translation-json-files

### PR DESCRIPTION
**Pull request by olehoef**
I restructured the json files by page (and some times components). I also updated all the keys in the code.

### CODE REVIEW

File: components/Navbar.vue [hunk 1]
```diff
@@ -123,7 +123,7 @@
         @click="seen = !seen"
       >
         <NuxtLink :to="localePath('/about') + '/'" class="header-link">{{
-          $t("about")
+          $t("nabbar.about")
```
Comment (jazzjacob) [inline components/Navbar.vue:126]: spelling mistake

File: pages/catalog/index.vue [hunk 2]
```diff
@@ -1,12 +1,12 @@
-<template>
+img<template>
```
Comment (jazzjacob) [inline pages/catalog/index.vue:1]: "img" mistake

**Review by jazzjacob — CHANGES_REQUESTED** [review]
fix the mistakes I've highlighted.

also there are inconsistencies with naming, sometimes camelCase, sometimes kebab-case, sometimes lowercase etc. but this is not urgent and we could start a new issue instead

**Comment by olehoef** [comment]
> fix the mistakes I've highlighted.
> 
> also there are inconsistencies with naming, sometimes camelCase, sometimes kebab-case, sometimes lowercase etc. but this is not urgent and we could start a new issue instead

OK thanks! The different key naming conventions still from the previous years. I would say let's do it later.

**Pull request merged**